docs/user_guide.md

```diff
@@ -8,42 +8,42 @@ The initiative aims to promote a free and open internet.
 ## Introduction
 
 The platform is divided into thematic categories called magazines. By default, any user can create their own magazine
-and automatically become its owner. Then they receive a number of administrative tools that will help them personalize
+and automatically become its owner. Then they receive many administrative tools that will help them personalize
 and moderate the magazine, including appointing moderators from among other users.
 
 Content from the Fediverse is also cataloged based on groups or tags. A registered user can follow magazines, other
 users or domains and create his own personalized homepage. There is also the option to block unwanted topics.
 
 Content can be posted on the main page - external links and more relevant articles or on microblog section - aggregating
-short posts. All content can be additionally categorized and labeled. Great possibilities to search for interesting
+short posts. All content can additionally be categorized and labeled. Great possibilities to search for interesting
 topics and people easily is something that distinguishes Kbin.
 
-Platform is equally suitable for a small personal instance for friends and family, a school or university community,
+The platform is equally suitable for a small personal instance for friends and family, a school or university community,
 company platform or a general instance with thousands of active users.
 
 ## User guide
 
 ### Customization
 
-Everyone has the ability to customize the appearance to suit your preferences. In the sidebar, you'll find an options
+Everyone can customize the appearance to suit their preferences. In the sidebar, you'll find an option
```

Comment:
"options" is more correct here, as the button's purpose is to reveal "a variety of settings". If the button used text rather than just an icon, it would more likely say "Options" rather than "Option".